server/src/main/xsd/gateway-config-201616.xsd

```diff
@@ -1490,6 +1550,55 @@
         </complexContent>
     </complexType>
 
+	<complexType name="turn.restServicePropertiesType">
+		<complexContent>
+			<restriction base="gateway:ServicePropertiesType">
+				<all>
+					<element maxOccurs="1" minOccurs="0" name="key.alias"
+						type="gateway:CollapsedString">
+						<annotation />
+					</element>
+					<element maxOccurs="1" minOccurs="0" name="key.password-file"
+						type="gateway:CollapsedString">
```

Comment:
The additional key password-file here did not pass acceptance testing via Robin and John.  The reasoning is is that we don't support securing specific keys in any of the other services.  If we do ever support that then we may solve it in a more generic way.  Thus we will remove this for now and always require the password to be the same...